contribution_process.md

```diff
@@ -0,0 +1,149 @@
+1. The number of users in the community that request the change, measured
+   through GitHub :+1: reactions.
+1. Design discussions, including use cases, edge cases, and alternative
+   solutions.
+
+Maintainers regularly review open GitHub issues and label the ones that are
+ready for community contribution with the `community` label and add them to the
+"Ready to contribute" lane of the project board.
+
+The decision to label an issue as ready for community contribution is ultimately
+up to the discretion of the maintainers. Depending on the issue type (bug or
+feature request) and area (code or documentation), the maintainers use a
+set of criteria to decide if the issue is ready for contribution. This may
+include:
+
+1. The change is feasible for a community member to contribute.
+1. Feature requests should have at least 10 :+1: reactions.
+1. The maintainers agree with the implementation plan, including any design
+   discussions (e.g., public interface design, edge cases, backwards
+   compatibility concerns).
+
+If the criteria are not satisfied, the maintainers should explicitly comment
+what criteria are not satsified and how the community can satisfy them. All
+discussions must be resolved prior to making a contribution so that feedback
+during the review process focuses on the implementation of the agreed upon
+design.
+
+Issues that do not receive enough :+1: reactions and do not have any activty for
+a year are considered stale. Without further interaction, they will be closed.
+
+## Contribution-ready stage
+
+A GitHub issue enters the contribution ready stage when a maintainer adds the
+issue to the "Ready to contribute" lane of the public [GitHub project
+board](https://github.com/orgs/aws/projects/21). This stage indicates any user
+is welcome to initiate a contribution via a GitHub pull request.
+
+## Implementation stage
+
+A GitHub issue enters the implementation stage once a user submits a GitHub pull
+request implementing the change. This stage indicates that a contribution has
+been submitted for the issue, but the maintainers have not committed to a full
+review of the contribution. The issue will be moved to the "Implementation" lane
+of the [GitHub project board](https://github.com/orgs/aws/projects/21). The
+maintainers will respond to let the contributor know to expect a preliminary
+review to confirm:
+
+1. It is implemented in the manner described in the issue.
+1. It meets the criteria specified in the contribution guide.
+
+If the criteria are not satisfied, the maintainers will comment on the issue
+specifically indicating what work is still required. If the contributor has
+questions, they should refer to this guide or ask the maintainers to clarify.
+Once the contributor makes the required changes, the maintainers will follow up.
+
+If the contributor is no longer actively engaged by responding to comments from
+maintainers or making code changes for more than 30 business days, the
```

Comment:
30 business days is going to be hard to automate. Should we just commit to a flat 30 calendar days or increase that number if longer is required?